pkg/processor/native/utils.go

```diff
@@ -14,14 +16,22 @@ func isOpaque(im image.Image) bool {
 	}
```

Comment:
What if struct passed is `RGBA` from `image` package which has an `Opaque` method, do we want to ignore that method and use our own opacity check?

Reply:
actually I was looking to that also, but for non-RGBA images (i.e: Gray images), their implementation is actually faster than ours, in Gray's case it always returns true.
Ideal scenario would be to rewrite all the Opaque() method for each image type so that it's more optimized but I think that would be overkill, wdyt?

Reply:
I think we can just check with a typecast to `image.Gray` since it doesn't have an Alpha channel, that's why they always return true.
Yes, rewriting all Opaque methods will be overkill.